devices/lixee.js

```diff
@@ -0,0 +1,475 @@
+];
+
+function getCurrentConfig(device, options) {
+    const endpoint = device.getEndpoint(1);
+
+    const linkyMode = (options.hasOwnProperty('linky_mode')? options['linky_mode'] : linkyModeDef.legacy);
+    const linkyPhase = (options.hasOwnProperty('energy_phase')? options['energy_phase'] : linkyPhaseDef.single);
+    const linkyProduction = (options.hasOwnProperty('production')? options['production'] : false);
+
+    // Main filter
+    let myExpose = exposedData
+        .filter((e) => e.linkyMode == linkyMode && (e.linkyPhase == linkyPhase || e.linkyPhase == linkyPhaseDef.all) && e.linkyProduction == linkyProduction);
+
+    // Filter even more, based on our current tarif
+    if (linkyMode == linkyModeDef.legacy) {
+        // Remove atributes which doesn't match current tarif
+        const currentTarf = endpoint.clusters.liXeePrivate.attributes.currentTarif.replace(/\0/g, '');
+        switch (currentTarf) {
+        case 'BASE':
+            myExpose = myExpose.filter((a) => ![
+                'HCHC',
+                'HCHP',
+                'HHPHC',
+                'EJPHN',
+                'EJPHPM',
+                'BBRHCJB',
+                'BBRHPJB',
+                'BBRHCJW',
+                'BBRHPJW',
+                'BBRHCJR',
+                'BBRHPJR',
+                'DEMAIN',
+                'PEJP'].includes(a.exposes.name));
+            break;
+        case 'HC..':
+            myExpose = myExpose.filter((a) => ![
+                'BASE',
+                'EJPHN',
+                'EJPHPM',
+                'BBRHCJB',
+                'BBRHPJB',
+                'BBRHCJW',
+                'BBRHPJW',
+                'BBRHCJR',
+                'BBRHPJR',
+                'DEMAIN',
+                'PEJP'].includes(a.exposes.name));
+            break;
+        case 'EJP.':
+            myExpose = myExpose.filter((a) => ![
+                'BASE',
+                'HCHC',
+                'HCHP',
+                'BBRHCJB',
+                'BBRHPJB',
+                'BBRHCJW',
+                'BBRHPJW',
+                'BBRHCJR',
+                'BBRHPJR',
+                'DEMAIN'].includes(a.exposes.name));
+            break;
+        case currentTarf.startsWith('BBR'):
+            myExpose = myExpose.filter((a) => ![
+                'BASE',
+                'HCHC',
+                'HCHP',
+                'EJPHN',
+                'EJPHPM',
+                'PEJP'].includes(a.exposes.name));
+            break;
+        default:
+            break;
+        }
+    }
+
+    return myExpose;
+}
+
+const definition = {
+    zigbeeModel: ['ZLinky_TIC'],
+    model: 'ZLinky_TIC',
+    vendor: 'LiXee',
+    description: 'Lixee ZLinky',
+    fromZigbee: [seMeteringFZ, haMeterIdentificationFZ, haElectricalMeasurementFZ, seLixeePrivateFZ],
```

Comment:
Why are there custom converters for metering and electrical measurement? (I would expect `fz.metering` and `fz.electrical_measurement` here)